### DISCUSSION TITLE
[HELP] Can someone teach me how to DOUBLE the aimbot from Osiris?

### DISCUSSION THREAD

**Discussion by diekunstderfuge** [danielkrupinski/Osiris#3979] [Q&A]
I know nothing about coding, I just know how to compile Osiris. 

I just want to have the option of having TWO AIMBOTs on the Osiris menu, so I can configure them differently and activate them with different keys (so they won't conflict).

It seems like a relatively simple copy/paste thing, but I don't know how to do that.

Can you help me?

**Comment by fanat1cc** [comment]
You would need to know how to change and recode the config saving and loading and know how to change the new aimbot to use different values and path names to the original and how to change the imgui code.

For a beginner, I would suggest looking into learning how to code, there are videos explaining how osiris works and how to code with things like I'm gui

Essentially since you have no prior experience, not to sound like an ass but your probably not going to do it without learning first
